### PR TITLE
news added

### PR DESCRIPTION
**Pull request by danieledebernardinDTD**
## Description

As already mentioned in #ISSUE_NUMBER, this PR tackles:
* ...
* ...
* ...

In particular, the ...

## Checklist

- [ ] I followed the indications in the [CONTRIBUTING](CONTRIBUTING.md)
- [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?
- [ ] Ready for review! :rocket:

## Fixes

- Fixes #

### CODE REVIEW

**Comment by danieledebernardinDTD** [comment]
cosa ho rotto @bfabio @westcoast ?

File: contents/home-page/home.yml [hunk 1]
```diff
@@ -130,46 +130,44 @@ opportunity:
...
+    source: "Vai all'Avviso"
+    linkTo: 'https://areariservata.padigitale2026.gov.it/Pa_digitale2026_dettagli_avviso?id=a017Q00001HAufCQAT'
+  - id: 4
     category: Linee guida di asseverazione
     title: "Aggiornamento di luglio 2023"
     description: "Pubblicate le indicazioni per Piattaforma notifiche - SEND, integrate le informazioni per la Migrazione al cloud e siti/servizi"
     source: "innovazione.gov.it"
     linkTo: 'https://innovazione.gov.it/notizie/articoli/pa-digitale-2026-le-novita-nelle-linee-guida-nell-aggiornamento-di-luglio'
-  - id: 2
+  - id: 5
```
Comment (bfabio) [inline contents/home-page/home.yml:157]:
```suggestion
  - id: 5 
```

Comment (bfabio) [reply contents/home-page/home.yml]: Bogus change to force rebuild

File: contents/home-page/home.yml [hunk 2]
```diff
@@ -130,46 +130,44 @@ opportunity:
...
```
Comment (bfabio) [inline contents/home-page/home.yml:157]:
```suggestion
  - id: 5
```

Comment (bfabio) [reply contents/home-page/home.yml]: Revert bogus change

**Pull request merged**